canvas_modules/common-canvas/src/common-properties/common-properties.jsx

```diff
@@ -266,7 +268,9 @@ CommonProperties.propTypes = {
 		validationHandler: PropTypes.func,
 		titleChangeHandler: PropTypes.func,
 		propertiesActionLabelHandler: PropTypes.func,
-		tooltipLinkHandler: PropTypes.func
+		tooltipLinkHandler: PropTypes.func,
+		propertyIconHandler: PropTypes.func,
+		iconCallBack: PropTypes.func
```

Comment:
Why do we have 2 new callbacks?

Reply:
Forgot to remove this one from previous commit